source/query-bar.txt

```diff
@@ -8,36 +8,36 @@ From the various Collections tabs, you can use the query bar to specify
 MongoDB queries.  For details, see :ref:`querying`.
 
 .. figure:: /images/compass/query-bar-documents-view.png
-   :figwidth: 780px
+   :figwidth: 805px
 
-You can specify a query filter in the query bar. If you click the 
+You can specify a query filter in the query bar. If you click 
 :guilabel:`Options` to expand the query bar, you also have access to 
 the ``project``, ``sort``, ``skip``, and ``limit`` options of the query.
 
 .. list-table::
    :widths: 15 85
 
-   * - ``Filter``
+   * - Filter
 
      - Filter for the schema view.
 	 
-   * - ``Project``
+   * - `Project <https://docs.mongodb.com/manual/reference/operator/aggregation/project/>`_
```

Comment:
@rob-mongodb  -- So, I wanted you to review the sections:
1) to see that aggregation and find operations are two separate things.   I believe these are project/sort/limit ... associated with find operations.
2) to ensure that you are familiar with the two (not in the abstract but in the concrete sense) -- so that you know how to use them, that you recognize they are separate things, etc.